tools/nni_cmd/launcher.py

```diff
@@ -360,9 +360,13 @@ def launch_experiment(args, experiment_config, mode, config_file_name, experimen
         module_name = AdvisorModuleName.get(package_name)
     if package_name and module_name:
         try:
-            check_call([sys.executable, '-c', 'import %s'%(module_name)], stdout=PIPE, stderr=PIPE)
+            stdout_full_path, stderr_full_path = get_log_path(config_file_name)
+            with open(stdout_full_path, 'a+') as stdout_file, open(stderr_full_path, 'a+') as stderr_file:
```

Comment:
why use `a+` rather than 'w'?

Reply:
the stdout_file and stderr_file is a common file, will also be used in other place of nnictl. `w` will override the content.